docker/caldera/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+caldera:
+  image: soarca-caldera
+  build: https://github.com/mitre/caldera.git
```

Comment:
Has been changed; we now use the Git thing everywhere

Reply:
also, what is the difference, eventually, with this file and the one added as `docker/soarca/docker-compose.caldera.yml`? Can we not remove one and work with the other only?

Reply:
> What does the `image` property do here? [...]

It peacefully sits there, but mainly guarantees the image name/tag to be consistent. Also, if the image happens to be locally available, it'll use it, though I don't expect that to be  the case ever.

> Can we not remove one and work with the other only?

Yeah, this file exists for spek en bonen, but I kept it just to keep consistency with the rest of the Docker setup... I'd prefer this consistency over getting rid of the file.